src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Placement } from '@popperjs/core';
+
+import UserAvatar from '~shared/UserAvatar';
+import ListItem from '~shared/ListItem';
+import { ListItemStatus } from '~shared/ListItem/ListItem';
+import { FormattedAction } from '~types';
+import { useColonyContext } from '~hooks';
+
+import { getActionListItemTitleValues, ActionsListItemMeta } from '.';
+
+const displayName = 'common.ColonyActions.ActionsListItem';
+
+const userAvatarPopoverOptions = {
+  showArrow: false,
+  placement: 'bottom-start' as Placement,
+  modifiers: [
+    {
+      name: 'offset',
+      options: {
+        offset: [-20, 10],
+      },
+    },
+  ],
+};
+
+// const isFullyNayStaked = (totalNayStake?: string, requiredStake?: string) =>
+//   BigNumber.from(totalNayStake || 0).gte(BigNumber.from(requiredStake || 0));
+
+// const shouldDisplayCountDownTimer = (motionId, motionState) => {
+//   const isMotionFinished =
+//     motionState === MotionState.Passed ||
+//     motionState === MotionState.Failed ||
+//     motionState === MotionState.FailedNoFinalizable;
+
+//   return motionId && !isMotionFinished;
+// };
+
+interface Props {
+  item: FormattedAction;
+}
+
+const ActionsListItem = ({
```

Comment:
There are two parts to this refactor. First is the abstraction of the view `ListItem`, which will also be shared by `DecisionListItem`.

The second is handling the logic, most of which was dedicated to formatting the title correctly. This is now handled by `getActionListItemTitleValues`.

The commented out sections refer to motions functionality.